gnovm/stdlibs/std/banker.gno

```diff
@@ -59,6 +61,9 @@ func (b BankerType) String() string {
 	}
 }
 
+// regexp for denom format
+var reDenom = regexp.MustCompile(`^[a-z][a-z0-9]{2,15}$`)
```

Comment:
can we actually make a simple helper function here instead? i don't think we should use regexp in "hot code" like this

Reply:
[903e06e](https://github.com/gnolang/gno/pull/3239/commits/903e06efb14e905d2620eaaace69f4f8249654da)